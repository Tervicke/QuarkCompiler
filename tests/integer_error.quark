//assigment errors
int x = "hel"
int x = 3.14
int x = false

//assig variables error
string str = "hello"
bool b = false
double f = 3.14

int x = str
int x = b
int x = f
