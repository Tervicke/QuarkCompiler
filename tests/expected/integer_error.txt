[31m[ERROR] Line 2:1 cannot assign string to an int[0m
[31m[ERROR] Line 3:1 cannot assign double to an int[0m
[31m[ERROR] Line 4:1 cannot assign bool to an int[0m
[31m[ERROR] Line 11:1 cannot assign string to an int[0m
[31m[ERROR] Line 12:1 cannot assign bool to an int[0m
[31m[ERROR] Line 13:1 cannot assign double to an int[0m
